basic_integral/basic_integral.cpp

```diff
@@ -0,0 +1,25 @@
+#include <iostream>
+#include <cmath>
+using namespace std;
+
+int main() {
+	int N = 10, b, i;
+	double sum, fx;
+	double p = 2*acos(0.0);
+	
+	cout << "How many iterations?" ;
+	cin >> b ;
+	
+	N = 10 * pow(b-1,2);
+	
+	for(int i=0; i<N; i++){

```

Comment:
I guess the problem statement was not clear enough and has probably confused you. If the user has entered for example 4 as the number of iterations (variable b), your code needs to calculate and print the result for N=10, 20, 40, 80. So you need a for loop over iterations as well!